### ISSUE TITLE
singly Linkedlist code 

### ISSUE THREAD

**Issue by Nickranjan** [loveBabbar/CodeHelp-DSA-Busted-Series#377]
singly linked list code has extra stuff which seems confusing , please bhaiya type the code only for the current lecture what you taught, no advance topics codes for the current lecture code.

**Comment by nova042** [comment]
can I contribute on  this issue

**Comment by AdhishthanAshok** [comment]
> can I contribute on this issue

Yeah ..anyone can contribute . Make sure to contribute if you have good options.
